### PR TITLE
feat(options): add prettierLast option for prettier-eslint

### PR DESCRIPTION
**Pull request by spudly**
Adds a new `Run Prettier Last` option (see prettier/prettier-eslint#58 and prettier/prettier-eslint#59).

### CODE REVIEW

**Comment by robwise** [comment]
@spudly This is great code, could you just add `// $FlowFixMe` before line 68 so it passes flow and then we can merge?

File: src/executePrettier.test.js [hunk 1]
```diff
@@ -60,6 +60,22 @@ describe('executePrettierOnBufferRange()', () => {
     expect(prettierEslint).toHaveBeenCalledWith({ filePath: 'foo.js', text: 'untransformed text' });
   });
 
+  test('passes prettierLast option to prettier-eslint', () => {
+    // $FlowFixMe
+    helpers.shouldUseEslint.mockImplementation(() => true);
+    // $FlowFixMe
+    helpers.getCurrentFilePath.mockImplementation(() => 'foo.js');
+    helpers.getPrettierEslintOptions.mockImplementation(() => ({ prettierLast: true }));
```
Comment (robwise) [inline src/executePrettier.test.js]: we just need to add a `// $FlowFixMe` here to pass flow

**Comment by spudly** [comment]
done.

File: src/executePrettier.test.js (hunk 1 above)
Comment (spudly) [inline src/executePrettier.test.js]: done.

**Comment by robwise** [comment]
@spudly awesome, can you squash down that commit?

**Comment by spudly** [comment]
squashed. also added myself as contributor the right way, as I had done it manully before.

**Pull request merged**